meeshkan/serve/mock/rest.py

```diff
@@ -37,10 +39,13 @@ def spew(
         for endpoint in self._endpoints:
             res = requests.post(endpoint, json={"request": req, "schemas": cs})
             cs = res.json()
-        return [
-            OpenAPISpecification(convert_to_openapi(api), source)
-            for source, api in cs.items()
-        ]
+
+        out: List[OpenAPISpecification] = []
+        for name, dict_spec in cs.items():
+            spec = convert_to_openapi(dict_spec)
+            storage_manager.add_mock(name, spec)
```

Comment:
This is also a global side-effect so it should somehow be made explicit somehow I think.

Reply:
Ugh, I keep missing these in my reading. Is there a way to lint for this in `isort` ?